src/app/core/permissions/ability/ability.service.ts

```diff
@@ -74,10 +75,20 @@ export class AbilityService extends LatestEntityLoader<Config<DatabaseRules>> {
     return this.interpolateUser(rawUserRules, sessionInfo);
   }
 
-  private interpolateUser(
+  private async interpolateUser(
     rules: DatabaseRule[],
-    user: SessionInfo,
-  ): DatabaseRule[] {
+    sessionInfo: SessionInfo,
+  ): Promise<DatabaseRule[]> {
+    const user = await this.entityMapper
+      .load(User, sessionInfo.entityId)
+      .catch(() => {});
```

Comment:
Didn't we deprecate the `User` entity and instead are using `SessionInfo` and for business logic the `CurrentUserSubject` to allow any entity type as the "profile" of a user account?

Reply:
switched to CurrentUserSubject